tests/lisa/test_trace.py

```diff
@@ -78,3 +78,28 @@ def test_setTaskName(self):
         self.assertEqual(trace.getTaskByName('father'), [1234])
 
         os.remove(self.test_trace)
+
+    def test_time_range(self):
+        """
+        TestTrace: time_range is the duration of the trace
+        """
+        expected_duration = 6.676497
```

Comment:
Not too sure if there's a better way to store this value. We could use ftrace.get_duration() but that's how time_range is set so I don't really see the relevance...